ErsatzTV/client-app/src/views/FFmpegProfilesPage.vue

```diff
@@ -27,10 +40,54 @@ export default class FFmpegProfiles extends Vue {
                 value: 'resolution'
             },
             { text: this.$t('ffmpeg-profiles.table.video'), value: 'video' },
-            { text: this.$t('ffmpeg-profiles.table.audio'), value: 'audio' }
+            { text: this.$t('ffmpeg-profiles.table.audio'), value: 'audio' },
+            { text: 'Actions', value: 'actions', sortable: false }
         ];
     }
 
+    addRecord() {
+        this.$router.push({
+            name: 'add-ffmpeg-profile'
+        });
+    }
+
+    deleteRecord(record: any, recordName: any) {
+        this.$swal
+            .fire({
+                title: 'Are you sure?',
+                text: 'Delete "' + recordName + '" FFmpeg Profile?',
+                icon: 'warning',
+                iconColor: '#4CAF50',
+                showCancelButton: true,
+                confirmButtonText: 'Yes'
+            })
+            .then((result) => {
+                if (result.isConfirmed) {
+                    let index = this.ffmpegProfiles.findIndex(
+                        (it) => it.id === record
+                    );
+                    this.ffmpegProfiles.splice(index, 1);
+                    ffmpegProfileApiService.deleteRecord(String(record));
+                    this.$swal.fire({
+                        html: '"' + recordName + '" FFmpeg Profile deleted.',
+                        timer: 2200
+                    });
+                    this.$swal.fire(
```

Comment:
Can we use a different form of the alert here so it doesn't require dismissing with a mouse click?

Reply:
Changed the delete confirmation to a toast message, no need to have a big middle confirmation dialog after clicking yes.